NimicClientTest/Fixtures.cs

```diff
@@ -0,0 +1,843 @@
+﻿using System;
+namespace NimiqClientTest
+{
+    public class Fixtures
+    {
+        public Fixtures()
+        {
+        }
+
+        public static string Accounts()
```

Comment:
Similar to the Python client, also here it's only JSON results, no classes or structures that help a user to create this objects. I think this makes it much harder to use, no?

Reply:
I see the objects exist, done

Reply:
Yes, those json strings are used only in the tests, the are already predefined data structures used by all methods like in the recent changes made for the Python implementation.